daebbang-api/src/test/resources/application-test.yml

```
@@ -71,12 +82,7 @@ spring:
-      region:
-        static: ap-northeast-2
-      credentials:
-        access-key: test
-        secret-key: test
-      s3:
-        enabled: false
\ No newline at end of file
+toss:
+  payments:
+    secret-key: test_toss_secret_key
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)